_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,256 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+The rule applies to each [`<meta>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element) element with a [`name`](https://www.w3.org/TR/html52/document-metadata.html#element-attrdef-meta-name) attribute whose value is set to [`viewport`](https://www.w3.org/TR/css-device-adapt-1/#viewport-meta) and has a [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute.
+
+## Expectation
+
+The [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute, whose value is mapped to a list of property/value pairs in a user-agent specific manner, does not:
```

Comment:
Expectations is disconnected from the applicability. That's why expectations start with "Each test target ..."